content/en/docs/languages/net/exporters.md

```diff
@@ -105,12 +108,35 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     // Other setup code, like setting a resource goes here too
 
     .Build();
+
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
```

Comment:
It'd be good to get a follow up PR to remove the pattern of "using var meterProvider", and prefer an explicit dispose().. We have a lot of people copying this to helper functions, and accidently causing it to get disposed() when helper method exits!

https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/getting-started-console/Program.cs#L27-L29

Separate PR to not further delay this PR